config/routes.rb

```diff
@@ -6,4 +6,13 @@
 
    get '/profile', to: "users#show"
    resources :users, only: [:new, :create]
+   get "/obituaries", to: "obituaries#index"
+   namespace :obituaries do
+     get 'covid-19', to: 'covid#index'
+     get 'recent', to: 'recent#index'
+   end
+   get '/search', to: 'search#index'
+   namespace :search do
```

Comment:
anyone know a cool way to do a 2-for-1 here, by creating the root controller with methods and then nesting the namespaced controllers?

Reply:
did you try get "/"?

Reply:
I think there is a way, but it will take a bit of digging, I will let you know if I see something